src/digitalIdentity-frontend/src/app/pages/change-password-page/change-password-page.component.html

```diff
@@ -1,4 +1,13 @@
 <mat-card class="change-password-card">
+  <div class="logos">
+    <img class="didentity_login" src="../../assets/images/DIdentity.png" alt="DIdentity" />
+
+    <img
+      class="adorsys_login"
+      src="../../assets/images/adorsys.png"
+      alt="adorsys"
+    />
+  </div>
```

Comment:
I think it would be better to separate the logo from the cards for the login and the change password

Reply:
The problem is that then there would be a white box around each of the logos which looks quite awful. I tried it but it looks better inside the cards.

Reply:
OK I understand. But would if we place a separator between the logo and the title of the card it would be a little better?

Reply:
I do not think so

Reply:
Okay!